src/nft-card-front.ts

```diff
@@ -91,20 +91,29 @@ export class NftCardFrontTemplate extends LitElement {
         text-align: left;
       }
       .asset-detail-price {
-        text-align: right;
-      }
-      .asset-detail-price {
+        align-items: flex-end;
         font-size: 18px;
         font-weight: 400;
         display: flex;
         flex-flow: row;
         justify-content: flex-end;
-        align-items: baseline;
+        line-height: 15px;
+        text-align: right;
+        padding: 6px 0;
       }
       .asset-detail-price img {
-        margin-left: 5px;
+        margin: 0 4px;
+      }
+      .asset-detail-price-current img {
         width: 15px;
-        align-self: center;
+      }
+      .asset-detail-price-previous {
+        font-size: 14px;
+        color: rgb(130, 130, 130);
+        line-height: 10px;
+      }
+      .asset-detail-price-previous img {
+        width: 1ex;
```

Comment:
woah, this looks like a rare unit - any reason? if not, let's stick to vw or rem?